Neuro/Pathfinding/DataStructures/PlatformNode.cs

```diff
@@ -27,4 +24,15 @@ public override bool IsTransportActive
     }
     private Vector3 _lastPosition;
     private bool _lastResult;
+
+    public override Node Clone()
+    {
+        return new PlatformNode(WorldPosition, GridPosition, IsAccessible, _platform)
+        {
+            Color = Color,
+            TransportSelfId = TransportSelfId,
+            TransportTargetId = TransportTargetId,
+            TransportNeighborsCache = TransportNeighborsCache
```

Comment:
Funny story, there's another Clone method
```suggestion
            TransportNeighborsCache = new List<Node>(TransportNeighborsCache)
```